benchmarks/src/Main.elm

```diff
@@ -0,0 +1,68 @@
+module Main exposing (main)
+
+import Benchmark exposing (Benchmark, describe)
+import Benchmark.Runner exposing (BenchmarkProgram, program)
+import Csv.Parser as Parser
+
+
+naiveParser : String -> List (List String)
+naiveParser =
+    String.split "\u{000D}\n" >> List.map (String.split ",")
+
+
+triplesCsv : Int -> String
+triplesCsv howManyRows =
+    List.range 0 howManyRows
+        |> List.map (String.fromInt >> List.repeat 3 >> String.join ",")
+        |> String.join "\u{000D}\n"
+
+
+naive : Benchmark
+naive =
+    [ 0, 1, 2, 4, 8 ]
+        |> List.map
+            (\size ->
+                let
+                    csv =
+                        triplesCsv size
+                in
+                ( String.fromInt size ++ " rows"
+                , \_ -> naiveParser csv
+                )
+            )
+        |> Benchmark.scale "naive"
+
+
+parser : Benchmark
+parser =
+    let
+        config =
+            case Parser.config { rowSeparator = "\u{000D}\n", fieldSeparator = "," } of
+                Ok config_ ->
+                    config_
+
+                Err _ ->
+                    crashButWithoutDependingOnDebug ()
```

Comment:
Would it be nice to export a default config using new lines and commas? So that in the most frequent use case folks don't have to do this.

Reply:
I think the most frequent case is `Decode.decodeCsv`, where it's already handled